### ISSUE TITLE
Incloure un apartat de qui som

### ISSUE THREAD

**Issue by theOfficeCat** [hwfib/hwfib.github.io#3]
Crec que podria ser una bona idea inclure un apartat de qui som els esutdiants que mantenim el lloc web amb links a les webs personal. Què us sembla?

**Comment by rdvdev2** [comment]
M'agrada l'idea, +1

**Comment by Serikpg** [comment]
No va en la línia de "el projecte"?? Li podem canviar el nom...

**Comment by rdvdev2** [comment]
Sí, crec que la solució a això és fer el canvi de "El projecte" a "Qui som?" o "Sobre nosaltres". Ho podem arreglar després de resoldre #5

**Comment by Serikpg** [comment]
Al final no es va fer res a aquest apartat oi?
Ara a l'estiu encara ens hi podriem possar
